docs: add vue devtools option (#8150)
withastro/docs: src/content/docs/en/guides/integrations-guide/vue.mdx

  ],
});
```

### devtools

> You can enable [`vuejs/devtools-next`](https://github.com/vuejs/devtools-next) in development by setting `devtools: true`.

Reviewer: ```suggestion
You can enable [Vue DevTools](https://devtools-next.vuejs.org/) in development by setting `devtools: true`.
```
What do you think about linking to the docs/marketing site? Going to GitHub directly doesn't tell you very much, and its own README just points here for even installation and setup.

  ],
});
```

### devtools

<p><Since v="4.2.0" /></p>

> You can enable [Vue DevTools](https://devtools-next.vuejs.org/) in development by setting `devtools: true` in the `vue()` integration config:

Reviewer: ```suggestion
You can enable [Vue DevTools](https://devtools-next.vuejs.org/) in development by passing an object with `devtools: true` to your `vue()` integration config:
```

```js title="astro.config.mjs"
import { defineConfig } from 'astro/config';
import vue from '@astrojs/vue';

export default defineConfig({
  // ...
  integrations: [vue({ devtools: true })],

> });

Reviewer: ``````suggestion
});
```
``````

  ],
});
```

### devtools

> <p><Since pkg="vue" v="4.2.0" /></p>

Reviewer: ```suggestion
<p><Since pkg="@astrojs/vue" v="4.2.0" /></p>
```
